build_tools/cmake/build_riscv.sh

```diff
@@ -55,16 +56,36 @@ else
   mkdir -p "${BUILD_RISCV_DIR?}"
 fi
 
-# Configure riscv for 64-bit linux config, then build.
-"${CMAKE_BIN?}" -G Ninja -B "${BUILD_RISCV_DIR?}" \
-  -DCMAKE_TOOLCHAIN_FILE="$(realpath ${ROOT_DIR?}/build_tools/cmake/riscv.toolchain.cmake)" \
-  -DIREE_HOST_BINARY_ROOT="$(realpath ${BUILD_HOST_DIR?}/install)" \
-  -DRISCV_CPU=rv64 \
-  -DIREE_BUILD_COMPILER=OFF \
-  -DIREE_ENABLE_MLIR=OFF \
-  -DIREE_BUILD_TESTS=ON \
-  -DIREE_BUILD_SAMPLES=OFF \
-  -DIREE_BUILD_EMBEDDING_SAMPLES=ON \
-  -DRISCV_TOOLCHAIN_ROOT="${RISCV_TOOLCHAIN_ROOT?}" \
-  "${ROOT_DIR?}"
+echo "Build riscv target with the config of ${RISCV_CONFIG?}"
+if [[ "${RISCV_CONFIG?}" == "rv64" ]]; then
+  # Configure riscv for 64-bit linux config, then build.
+  "${CMAKE_BIN?}" -G Ninja -B "${BUILD_RISCV_DIR?}" \
+    -DCMAKE_TOOLCHAIN_FILE="$(realpath ${ROOT_DIR?}/build_tools/cmake/riscv.toolchain.cmake)" \
+    -DIREE_HOST_BINARY_ROOT="$(realpath ${BUILD_HOST_DIR?}/install)" \
+    -DRISCV_CPU="${RISCV_CONFIG?}" \
+    -DIREE_BUILD_COMPILER=OFF \
+    -DIREE_ENABLE_MLIR=OFF \
+    -DIREE_BUILD_TESTS=ON \
+    -DIREE_BUILD_SAMPLES=OFF \
+    -DIREE_BUILD_EMBEDDING_SAMPLES=ON \
+    -DRISCV_TOOLCHAIN_ROOT="${RISCV_TOOLCHAIN_ROOT?}" \
```

Comment:
I should've noticed this in the docker image update, but it seems a bit strange that this gets a generic name whereas RISCV_RV32_NEWLIB_TOOLCHAIN_ROOT is very specific. This one is specifically for rv64 linux, right?

Reply:
Yes it was. I was hesitating on changing the docker env variable for the old rv64-linux toolchain for I may break the CI. I will clean it up in a separate PR.